collect_app/src/test/java/org/odk/collect/android/draw/PenColorPickerDialogTest.kt

```diff
@@ -0,0 +1,39 @@
+package org.odk.collect.android.draw
+
+import android.graphics.Color
+import androidx.test.espresso.action.ViewActions.click
+import androidx.test.espresso.matcher.ViewMatchers.withText
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import org.hamcrest.CoreMatchers.`is`
+import org.hamcrest.MatcherAssert.assertThat
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.kotlin.mock
+import org.mockito.kotlin.verify
+import org.odk.collect.android.R
+import org.odk.collect.fragmentstest.DialogFragmentTest
+
+@RunWith(AndroidJUnit4::class)
+class PenColorPickerDialogTest {
+
+    @Test
+    fun `dialog should be cancelable`() {
+        val scenario = DialogFragmentTest.launchDialogFragment(PenColorPickerDialog::class.java)
```

Comment:
This should be using `FragmentScenarioRule`/`FragmentLauncherRule` now so that the scenarios are cleaned up properly.